test_data/input_data/config/sources_config.json

```diff
@@ -0,0 +1,256 @@
+{
+  "entities": {
+    "Individual": {
+      "attributes": [
+        {
+          "name": "individual_id",
+          "sources": [
+            {
+              "file": "individual.tsv",
+              "column": "individual_id"
```

Comment:
so this one doesn't have to be specified?

Reply:
Yes, for now it's just here for testing purposes.